crypto/ct/ct_sct.c

```diff
@@ -223,6 +237,54 @@ int SCT_set1_signature(SCT *sct, const unsigned char *sig, size_t sig_len)
     return 1;
 }
 
+int SCT_set1_serialized_signature(SCT *sct, const unsigned char **in,
```

Comment:
The size of the signature is the 3-4th bytes of *in, so the length isn't necessarily known in advance. In CT v1, the signature is the last thing in the SCT though, so you can assume the signature takes up the remainder of *in. Anyway, reverting...

By the way, while it's probably a bit late to consider, there's an alternative solution here that splits the signature out into its own struct and so the `i2o_SCT_signature` functions are at least consistent with the others then: https://github.com/RJPercival/openssl/commit/3ed6bb9315ff36fb6d110f9bdc36d3463cf064ca